core-bundle/tests/Security/Authentication/Token/FrontendPreviewTokenTest.php

```diff
@@ -56,9 +56,9 @@ public function testSerializesItself(): void
         $serialized = $token->__serialize();
 
         if (method_exists($token, 'getRoles')) {
```

Comment:
This should always be true in Symfony 5.4, so you might remove the if condition. Not strictly related to this PR though.

Reply:
Fixed in 1f3b2532f.